packages/e2e-tests/specs/local/demo.test.js

```diff
@@ -29,7 +29,11 @@ describe( 'new editor state', () => {
 				onRequestMatch: createJSONResponse( MOCK_VIMEO_RESPONSE ),
 			},
 		] );
-		await visitAdminPage( 'post-new.php', 'gutenberg-demo' );
+
+		await Promise.all( [
+			visitAdminPage( 'post-new.php', 'gutenberg-demo' ),
```

Comment:
So the issue is with `page.goto` inside `visitAdminPage`. Do you think that we should enforce using `networkidle0` for this URL as an alternative approach? Should we use `networkidle0` each time we load a WordPress page? I'm trying to better understand if this is just a very content heavy page causing issues or it's s general concern that is less annoying in other places.

Reply:
Sorry for taking so long to reply... 
I managed to get a little time, so I could be reproducing the exact case where #27229 would happen:

![navigation_bug](https://user-images.githubusercontent.com/18705930/100641516-51d2d200-3316-11eb-96aa-0597e4919bab.png)

According to the [puppeteer documentation](https://github.com/puppeteer/puppeteer/blob/main/docs/api.md#pagegotourl-options), the default `options.waitUntil` value is [`load`](https://developer.mozilla.org/en-US/docs/Web/API/Window/load_event) - which means that `page.goto` returned promise is resolved when the `load` event fires. In a specific case - which I didn't investigate any further; only reproduced it - the `load` event is fired, and after that, resources are loaded by the page, as you can see in the screenshot. This caused the #27229 bug. 

The `networkidle0`, [as documented](https://github.com/puppeteer/puppeteer/blob/main/docs/api.md#pagegotourl-options), "_considers navigation to be finished when there are no more than 0 network connections for at least 500 ms_". It can be a good solution to the case where requests are made after the `load` event is fired.

Conclusion: It could happen at any moment where the page fires the `load` event and performs requests after that, but I am not sure whether this could happen on other pages.

Reply:
We have everything documented. Let's wait a few days or weeks and see whether a similar error strikes again.